app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java

```diff
@@ -208,7 +208,11 @@ public void showOrderStatistics(OrderStatistics orderStatistics) {
 
     @Override
     public void showError(String error) {
-        ViewUtils.showSnackbar(binding.getRoot(), error);
+        if (error.equals("starts-at should be after current date-time - starts-at")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=4963535)